server/routerlicious/packages/services/src/throttler.ts

```diff
@@ -0,0 +1,92 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+    IThrottler,
+    IThrottlerHelper,
+    IThrottlerResponse,
+    ThrottlingError,
+    ILogger,
+} from "@fluidframework/server-services-core";
+import LRUCache from "lru-cache";
+
+/**
+ * A lenient implementation of IThrottlerHelper that prioritizes low latency over strict throttling.
+ * This should be used for implementing throttling in places where latency matters more than accuracy,
+ * such as service endpoints or socket connections.
+ */
+export class Throttler implements IThrottler {
+    private readonly lastThrottleUpdateAtMap: LRUCache<string, number>;
+    private readonly countDeltaMap: LRUCache<string, number>;
+    private readonly throttlerResponseCache: LRUCache<string, IThrottlerResponse>;
+
+    constructor(
+        private readonly throttlerHelper: IThrottlerHelper,
+        private readonly minThrottleIntervalInMs: number,
+        private readonly logger?: ILogger,
+        maxCacheSize: number = 1000,
+        maxCacheAge: number = 1000 * 60,
+    ) {
+        const cacheOptions: LRUCache.Options<string, any> = {
+            max: maxCacheSize,
```

Comment:
Nit: maxSize

Reply:
`max` is the name of the option defined by the `lru-cache` npm package
https://www.npmjs.com/package/lru-cache#options